document_page_approval_levels/views/document_page_view.xml

```diff
@@ -6,6 +6,38 @@
             <field name="model">document.page</field>
             <field name="inherit_id" ref="document_page.view_wiki_form"/>
             <field name="arch" type="xml">
+                <xpath expr="//field[@name='name']" position="after">
+                    <div class="oe_button_box oe_right">
+                        <field name="content_can_print" invisible="1"/>
+                        <button name="%(lodi_document.lodi_document_page_report)d"
```

Comment:
@LuisAlejandro lido_document already depends of all documents modules, please the changes concerning to the report do it directly on lodi_document module as I mentioned on my previous message "this part of the code must be directly on lodi_document module".